docs/usage/dependency-injection.rst

```diff
@@ -80,8 +80,8 @@ The pre-requisites for dependency injection are these:
 
 What is *scope* in this context? Dependencies are **isolated** to the context in which they are declared. Thus, in the
 above example, the ``local_dependency`` can only be accessed within the specific route handler on which it was declared;
-The ``controller_dependency`` is available only for route handlers on that specific controller; And the router
-dependencies are available only to the route handlers registered on that particular router. Only the ``app_dependencies``
+The ``controller_dependency`` is available only for route handlers on that specific controller; And the ``router
+dependencies`` are available only to the route handlers registered on that particular router. Only the ``app_dependencies``
```

Comment:
If this change is being done, wouldn't it make sense to use

```
``router_dependency`` is available only to the route handlers registered on that particular router. Only the 
```

Considering the other two are variables you can use in the code?

Reply:
That's a good point! It's been reflected.